packages/metrics/src/features/legacy/handler.ts

```diff
@@ -174,3 +179,79 @@ export async function generateApplicationStartedPoint(
     timestamp: `${parseInt(searchResult._source.createdAt, 10) * 1000000}`
   }
 }
+
+export async function generateLegacyEventDurationHandler(
```

Comment:
@euanmillar I didn't wrote any test case for this as it's written for updating production data once